### PR TITLE
HIGH_LATENCY message for Iridium connections

### PR DESCRIPTION
**Pull request by dagar**
First pass at creating a single message appropriate for Iridium connections.

Based on @LorenzMeier's proposal here - https://groups.google.com/forum/#!searchin/mavlink/iridium%7Csort:relevance/mavlink/hv7zJ5Xdwcc/lHhZ4lXXBwAJ

TODO:
 - EKF status?
 - bitmap for actuator health?
 - max size? Should we aim for 50 bytes?

### CODE REVIEW

File: message_definitions/v1.0/common.xml [hunk 1]
```diff
@@ -3570,6 +3570,37 @@
...
+            <field name="longitude" type="int32_t">Longitude, expressed as degrees * 1E7</field>
+            <field name="altitude_home" type="int16_t">Altitude above the home position (meters)</field>
+            <field name="altitude_amsl" type="int16_t">Altitude above mean sea level (meters)</field>
+            <field name="altitude_sp" type="int16_t">Altitude setpoint relative to the home position (meters)</field>
+            <field name="airspeed" type="uint8_t">airspeed (m/s)</field>
+            <field name="airspeed_sp" type="uint8_t">airspeed setpoint (m/s)</field>
+            <field name="groundspeed" type="uint8_t">groundspeed (m/s)</field>
+            <field name="climb_rate" type="int8_t">climb rate (m/s)</field>
+            <field name="gps_nsat" type="uint8_t">Number of satellites visible. If unknown, set to 255</field>
+            <field name="gps_fix_type" type="uint8_t">0-1: no fix, 2: 2D fix, 3: 3D fix, 4: DGPS, 5: RTK.</field>
```
Comment (WickedShell) [inline message_definitions/v1.0/common.xml]: This should use the GPS_FIX_TYPE enum.

Comment (dagar) [inline message_definitions/v1.0/common.xml]: Thanks, fixed.

**Comment by LorenzMeier** [comment]
Looks good to merge. I guess we should consider this one experimental but start using it to gain some experience.

File: message_definitions/v1.0/common.xml [hunk 2]
```diff
@@ -3570,6 +3570,37 @@
...
+            <field name="longitude" type="int32_t">Longitude, expressed as degrees * 1E7</field>
+            <field name="altitude_home" type="int16_t">Altitude above the home position (meters)</field>
+            <field name="altitude_amsl" type="int16_t">Altitude above mean sea level (meters)</field>
+            <field name="altitude_sp" type="int16_t">Altitude setpoint relative to the home position (meters)</field>
+            <field name="airspeed" type="uint8_t">airspeed (m/s)</field>
+            <field name="airspeed_sp" type="uint8_t">airspeed setpoint (m/s)</field>
+            <field name="groundspeed" type="uint8_t">groundspeed (m/s)</field>
+            <field name="climb_rate" type="int8_t">climb rate (m/s)</field>
+            <field name="gps_nsat" type="uint8_t">Number of satellites visible. If unknown, set to 255</field>
+            <field name="gps_fix_type" type="uint8_t">See the GPS_FIX_TYPE enum.</field>
```
Comment (vooon) [inline message_definitions/v1.0/common.xml]: `enum="GPS_FIX_TYPE"`

File: message_definitions/v1.0/common.xml [hunk 3]
```diff
@@ -3570,6 +3570,37 @@
            <field type="uint8_t" name="len">data length</field>
            <field type="uint8_t[180]" name="data">RTCM message (may be fragmented)</field>
         </message>
+        <message id="234" name="HIGH_LATENCY">
```
Comment (vooon) [inline message_definitions/v1.0/common.xml]: Why not to use 2.0 msgid?

**Pull request merged**